puppet/modules/web/manifests/init.pp

```diff
@@ -129,6 +131,24 @@
     max_connections => $max_rsync_connections,
   }
 
+  # DOCS
+  $docs_directory = '/var/www/vhosts/docs/htdocs'
+  $docs_directory_config = [
+    {
+      path            => $docs_directory,
+      options         => ['Indexes', 'FollowSymLinks', 'MultiViews'],
+      expires_active  => 'on',
+      expires_default => 'access plus 4 hours',
+    },
+  ]
+
+  $docs_attrs = {
+    servername   => 'docs.theforeman.org',
+    docroot      => $docs_directory,
+    docroot_mode => '2575',
```

Comment:
That's a weird mode. Why is it writable for the group but not the user? We have a special case with yum, but docs should be more like `website`. I'd also prefer a specific user that owns the docroot so we don't have to abuse root for it.

Reply:
Yeah I picked a really special example. Let me fix it in a new PR.